test/test_ad.jl

```diff
@@ -40,3 +40,18 @@ jac_cache = ForwardColorJacCache(f,x,color = repeat(1:3,10))
 forwarddiff_color_jacobian!(_J1, f, x, jac_cache)
 @test _J1 ≈ J
 @test fcalls == 1
+
+using BenchmarkTools
```

Comment:
a benchmark folder could be better to avoid putting this on CI every time

Reply:
This isn't a real PR.